components/cdc/src/service.rs

```diff
@@ -428,7 +444,8 @@ impl Service {
             };
             explicit_features = headers.features;
         }
-        info!("cdc connection created"; "downstream" => ctx.peer(), "features" => ?explicit_features);
+        info!("cdc connection created"; "downstream" => ctx.peer(),
+         "features" => ?explicit_features, "conn_id" => ?conn_id);
```

Comment:
set the feature as first field since downstream print the ip address, it's more important than the feature, which always print `multiplexing`